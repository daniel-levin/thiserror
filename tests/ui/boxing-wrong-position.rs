use thiserror::Error;

#[derive(Error, Debug)]
#[error("...")]
#[boxing]
pub struct AlsoMissing {
    #[from] err: std::io::Error,
}

fn main() {}
