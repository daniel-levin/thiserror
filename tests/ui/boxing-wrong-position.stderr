error: attribute #[boxing] expected on a specific field also having #[from] attribute
 --> tests/ui/boxing-wrong-position.rs:5:1
  |
5 | #[boxing]
  | ^^^^^^^^^
